apps/shared/avifexif.c

```diff
@@ -0,0 +1,63 @@
+// Copyright 2022 Google LLC. All rights reserved.
```

Comment:
`// Copyright 2022 Google LLC. All rights reserved.`

maybe we need a normalization pass here, I think this is now `// Copyright 2022 Google LLC`

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/1201.